_includes/head.html

```diff
@@ -10,3 +10,71 @@
   <link rel="canonical" href="{{ page.url | replace:'index.xml','' | prepend: site.baseurl | prepend: site.url }}" />
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
 </head>
+<style>
```

Comment:
Technically, `<style>` is only allowed as a child of `<head>` (see [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style) and elsewhere).

Reply:
Thanks, I always wonder about that. I'll move it.